release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -76,8 +76,14 @@ Because {product-title} {product-version} now uses a {op-system-base} 9.2 based
 
 [id="ocp-4-14-aws-shared-vpc"]
 ==== Installing a cluster on Amazon Web Services (AWS) using a shared VPC
+
 In {product-title} 4.14, you can install a cluster on AWS that uses a shared VPC, with a private hosted zone in a different account than the cluster. For more information, see xref:../installing/installing_aws/installing-aws-vpc.adoc#installing-aws-vpc[Installing a cluster on AWS into an existing VPC].
 
+[id="ocp-4-14-aws-s3-deletion"]
+==== Enabling S3 bucket to be retained during cluster bootstrap on AWS
+
+With this release users are able to opt out of the automatic deletion of the S3 bucket during the cluster bootstrap on Amazon Web Services. This is key for customers with security policies preventing the deletion of S3 buckets.
```

Comment:
One of the recommendation from the Red Hat Suppl guide for [short description](https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc) suggests:
> Use customer-centric language, such as "You can…​ by…​" or "To…​, configure…​".

```suggestion
With this update, you can opt out of the automatic deletion of the S3 bucket during the cluster bootstrap on AWS. This option is useful when you have security policies that prevent the deletion of S3 buckets.
```